en/docs/fragments/register-a-spring-boot-app.md

```diff
@@ -1,55 +1,24 @@
+
+4.	Provide the name for the service provider as `sample-app` and click **Register**. 
+
+5.	Now you will be redirected to the **Edit Service Provider** page.
+
+6.	Expand the  **Inbound Authentication Configuration** section and click **Configure** under the **OAuth/OpenID Connect Configuration** section.
+
+7.	Provide the callback URL as `http://localhost:8080/spring-boot-app/login/oauth2/code/wso2`.
+
+8.	Click **Update**.
+
+9.	Once the service provider is registered, you will be redirected to the **Service Provider Details** page. Here, expand the **Inbound Authentication 			Configuration** section and click the **OAuth/OpenID Connect Configuration** section. Copy the value of  `OAuth Client Key` shown here.
 
-      - http://localhost:8080/spring-boot-sample/login/oauth2/code/wso2 
-      - http://localhost:8080/spring-boot-sample/login
- 
-    !!!tip
-        The **Callback URL** is the exact location in the service provider's application where a authorization code
-        would be sent. This should be always `{baseUrl}/login/oauth2/code/wso2` .
-            
-           Example: http://localhost:8080/login/oauth2/code/wso2
-            
-        If you want configure oidc logout also, then you need to add post-logout-url.  You can add multiple callback
-        urls. In WSO2 Identity Server, post-logout-url can be configured via callback URL.
-    
-           Example:  http://localhost:8080/spring-boot-app/login
-           
-     
-
- 8. Click **Next**
- 
- 9. View the application details and Click **Finish**
- 
- 10. Click the **Access** tab and note the **Client ID** and **Client Secret** that appear. 
- 
-  
-| Field                 | Value                                                                 | 
-| --------------------- | ------------------------------                                        | 
-| Service Provider Name | your-application-name                                                 |
-| Description           | This is a spring-boot application                                     | 
-| CallBack Url          | {baseUrl}/login/oauth2/code/wso2,{baseUrl}/spring-boot-app/login      |
-                        
-
-**Eg:**
  
 | Field                 | Value                                                                                                 | 
 | --------------------- | -----------------------------                                                                         | 
 | Service Provider Name | sample-app                                                                                            |
 | Description           | This is a spring-boot application                                                                     | 
-| CallBack Url          | http://localhost:8080/sprinb-boot-app/login/oauth2/code/wso2, http://localhost:8080/spring-boot-app   |
+| CallBack Url          | http://localhost:8080/spring-boot-app/login/oauth2/code/wso2                                          |
```

Comment:
is it confirmed that `http://localhost:8080/spring-boot-app` is not needed?

Reply:
Yes, in 5.11.0 we have mentioned only one callback URL while providing instructions in terms of management console. https://is.docs.wso2.com/en/latest/develop/spring-boot/